### PR TITLE
Updated port range per #266 and add FAQ entry per #256

### PR DESCRIPTION
**Pull request by critzo**
This PR adds two updates on the FAQ page. First, it corrects the port ranges used by NDT referenced in #266 and adds an entry to mention where and how we host our servers, addressing #256

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/272)

### CODE REVIEW

File: _pages/faq.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ The graphic below illustrates the relationship between tests run by consumers an
 
 ![Illustration showing the lifecycle of an M-Lab test]({{ site.baseurl }}/images/about/lifecycle-of-mlab-test.png)
 
+## Where are M-Lab Servers hosted?
+{:.no_toc}
+
+M-Lab hosts our servers, which we call "pods", in Tier 1 data-centers, connected to the internet with links between 1-10 Gbps. In major metro areas, M-Lab strives to maintain multiple pods, each connected to as many distinct transit providers as possible.
```
Comment (georgiamoon) [inline _pages/faq.md]: Should we add a link here to the infrastructure page?

File: _pages/faq.md [hunk 2]
```diff
@@ -20,6 +20,13 @@ The graphic below illustrates the relationship between tests run by consumers an
...
+
+Once an M-Lab pod is online we make no changes in our systems for the life of each pod, but we don't control the routes and connectivity of the providers who connect our pods to the internet. That connectivity will change over time as one might expect, as upstream providers peer, upgrade capacity etc. Whenever there is a substantial change to our sites, such as renumbering of IP addresses, we retire the pod, retain its naming and metadata in our systems, and treat the pod as brand new.
```
Comment (georgiamoon) [inline _pages/faq.md]: This sentence `Once an M-Lab pod is online we make no changes in our systems for the life of each pod, but we don't control the routes and connectivity of the providers who connect our pods to the internet.` is a little confusing. Maybe rephrase?

`as one might expect` -- remove

Comment (critzo) [inline _pages/faq.md]: A rephrase:
`Once an M-Lab pod is online we make no changes in our systems for the life of the pod.`  --just dropping the part about routes to the pod...

File: _pages/faq.md (hunk 1 above)
Comment (critzo) [inline _pages/faq.md]: I was thinking that, but then didn't. Do you think we should add a menu link to it as well? Maybe under 'About'?

**Comment by critzo** [comment]
@georgiamoon I made an update based on your feedback.

File: _pages/faq.md (hunk 1 above)
Comment (georgiamoon) [inline _pages/faq.md]: I think just a link here is fine. I don't think under About makes sense, but I'd want to revisit the site architecture notes that we have, where we had a plan for where it goes. Let's keep that as out of scope of this PR

**Comment by georgiamoon** [comment]
@critzo should be good with a link to infrastructure in that first paragraph. LGTM.

File: _pages/faq.md (hunk 1 above)
Comment (critzo) [inline _pages/faq.md]: sgtm
